core/templates/pages/exploration-editor-page/feedback-tab/services/thread-data-backend-api.service.ts

```diff
@@ -165,11 +171,12 @@ export class ThreadDataBackendApiService {
         let feedbackThreadBackendDicts = threadData.feedback_thread_dicts;
         let suggestionThreadBackendDicts = threadData.suggestion_thread_dicts;
 
-        let suggestionBackendDictsByThreadId = new Map(
-          suggestionBackendDicts.map(dict => [
-            this.suggestionsService.getThreadIdFromSuggestionBackendDict(dict),
-            dict
-          ]));
+        let suggestionBackendDictsByThreadId:
+         Map<string | null, SuggestionBackendDict> = new Map(
```

Comment:
Wait, why can the key of the `Map` be `null`?

Reply:
In the comment above https://github.com/oppia/oppia/pull/13459/files#r683178393 if `dict` is null then key-value for `suggestionBackendDictsByThreadId` will be null, added a comment.

Reply:
Can we try to get rid of this somehow?

Reply:
Yep, I tried to remove this, but there were frontend test failures. We're not sure of the type of `dict` we're receiving. If it is null then null would be passed as a value and thus `null` is a type.